src/Components/Blazor/Build/src/targets/Blazor.MonoRuntime.targets

```diff
@@ -201,6 +205,14 @@
       <_DotNetHostFileName Condition=" '$(OS)' == 'Windows_NT' ">dotnet.exe</_DotNetHostFileName>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <_TypeGranularityLinkingConfig>$(BlazorIntermediateOutputPath)linker.typegranularityconfig.xml</_TypeGranularityLinkingConfig>
+    </PropertyGroup>
+    <GenerateTypeGranularityLinkingConfig Assemblies="@(_BlazorAssemblyToLink)" OutputPath="$(_TypeGranularityLinkingConfig)" />
```

Comment:
It might be ever so slightly nicer to filter in MSBuild rather than in the task

Reply:
```suggestion
    <GenerateTypeGranularityLinkingConfig Assemblies="@(_BlazorAssemblyToLink->WithMetadataValue('TypeGranularity', 'true'))" OutputPath="$(_TypeGranularityLinkingConfig)" />
```

Reply:
Sure, will do. Previously I thought it was more flexible to pass more info into the task in case it wanted to do something with the other assemblies, but since this is all internal, we're free to change it in the future if there's ever a reason to do so.

Right now I agree it's more direct and obvious to pass a pre-filtered list to the task.